src/apolo_app_types/protocols/common/hugging_face.py

```diff
@@ -23,9 +23,6 @@ class HuggingFaceModel(AbstractAppFieldType):
         description="The name of the Hugging Face model.",
         title="Hugging Face Model Name",
     )
-    model_files: str | None = Field(  # noqa: N815
```

Comment:
why dropping it?

Reply:
might we use it in the future? I suspect, we might

Reply:
I am just against empty fields that we don't use rn, cause it messes the UI